### ISSUE TITLE
View issues in 3.0 Beta on Windows

### ISSUE THREAD

**Issue by tkeiderling** [Zettlr/Zettlr#4052]
### Description

- The ctrl-j shortcut produces distraction free mode but does not mute paragraphs as expected (when the setting is turned on in preferences). Also, the "distraction free mode" leaves little bits of the normal view still showing at the top of the screen: 

<img width="277" alt="image" src="https://user-images.githubusercontent.com/90908474/212493523-3da20155-d26d-4a29-8689-7209592cd11d.png">

- The ctrl-alt-t shortcut does not produce typewriter mode but instead mutes all paragraphs except the one in focus. 
- In other words, a combination of ctrl-j and ctrl-alt-t produces what ctrl-j used to produce in 2.3.0. 

### Reproducing

- In the normal "editor" view on windows, press the shortcut ctrl-j or ctrl-alt-t. 

### Zettlr Version

Beta (if applicable)

### Specify version

3.0

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Windows 10

**Comment by nathanlesage** [comment]
Oh … umm, could be that I accidentally switched out the functionality … ¯\_(ツ)_/¯ 

I'll get to that!
